models/prices/bnb/prices_bnb_tokens.sql

```diff
@@ -112,7 +112,7 @@ FROM
     ('titano-titano', 'bnb', 'TITANO', 0x4e3cabd3ad77420ff9031d19899594041c420aee, 18),
     ('tlm-alien-worlds', 'bnb', 'TLM', 0x2222227e22102fe3322098e4cbfe18cfebd57c95, 4),
     ('tpt-token-pocket', 'bnb', 'TPT', 0xeca41281c24451168a37211f0bc2b8645af45092, 4),
-    ('tusd-trueusd', 'bnb', 'TUSD', 0x14016e85a25aeb13065688cafb43044c2ef86784, 18),
+    ('tusd-trueusd', 'bnb', 'TUSD', 0x40af3827F39D0EAcBF4A168f8D4ee67c121D11c9, 18),
```

Comment:
SCAN: https://bscscan.com/token/0x40af3827F39D0EAcBF4A168f8D4ee67c121D11c9

Reply:
However the original address may be still valid?

Reply:
The pegged token (0x14016...) is being deprecated but holders and still interact with the contract in order to migrate tokens and positions in defi to the new contract

I think the native token (0x40af38...) is preferable to the pegged token